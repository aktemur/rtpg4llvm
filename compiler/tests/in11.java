/*expr*/
 x[2] = 42  ;
