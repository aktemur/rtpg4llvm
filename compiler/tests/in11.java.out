mkBinOp(Equal,
         new ArrayExpr(new VarExpr("x"), mkConst(2)),
         mkConst(42))
